src/alpha-lab/multiple-rim.md

```diff
@@ -4,93 +4,166 @@ permalink: /alpha-lab/multiple-rim/
+    
+Check out these videos which show how we made the scans (also be sure to follow our [best practices](/enrichments/reference-image-mapper/#scanning-best-practices) for optimal scanning):
+
+::: tip
+The Reference Image Mapper prefers feature-rich environments. If you have large plain surfaces, like empty tables or countertops, 
+consider placing some strategic items within the environment to increase the chances of successful mapping. 
 :::
 
-## Run the enrichments
+<div style="display: flex;">
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0;">
+      <iframe style="position: absolute; width: 100%; height: 100%; border: none;" src="https://www.youtube.com/embed/jeL8gs053lg?si=6wlx4fjxlfiqrbRq"></iframe>
+    </div>
+  </div>
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0;">
+      <iframe style="position: absolute; width: 100%; height: 100%; border: none;" src="https://www.youtube.com/embed/zksTzVkGifk?si=3bxl0eKOgRbfoes-"></iframe>
+    </div>
+  </div>
+</div>
 
-Here we recorded just one video where the wearer was asked to walk and freely explore the living room. Now it is time to map the subject's gaze from this video into the five pictures above.
+<div style="display: flex;">
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0;">
+      <iframe style="position: absolute; width: 100%; height: 100%; border: none;" src="https://www.youtube.com/embed/Bg_SiFByceY?si=d2koC7-V7bbrYL3h"></iframe>
+    </div>
+  </div>
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0;">
+      <iframe style="position: absolute; width: 100%; height: 100%; border: none;" src="https://www.youtube.com/embed/0r8oAn2AZMQ?si=SbSVHedGTJ4Zshfw"></iframe>
+    </div>
+  </div>
+</div>
 
-<div class="iframe-container3">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/XTIkB8Wct6M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<div style="display: flex; justify-content: center;">
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0; margin-bottom: -35%;">
+      <iframe style="position: absolute; width: 50%; height: 50%; left: 50%; transform: translateX(-50%); border: none;" src="https://www.youtube.com/embed/fmy9F8Q9eW0?si=F7q399iZHGW2kArv"></iframe>
+    </div>
+  </div>
```

Comment:
Something off with the markup here. Try selecting text in point 3 below on staging and it will highlight/select the video. Which leads me to believe that the html markup is not entirely correct. Please check.